e2e/tests/check-browser-title.spec.ts

```diff
@@ -1,5 +0,0 @@
-import { expect, test } from "#e2e/fixtures.ts";
-
-test("has title", async ({ page }) => {
```

Comment:
This test is too expensive for what it does. A single test case requires about 5s of setup. It’s much easier to assert this in other tests